### PR TITLE
Replace signal with variable

### PR DESCRIPTION
**Pull request by andreas-junghanns**
Checklist

* [ ] Used a [personal fork](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo) of the repository to propose changes.
* [ ] [Built the specification](https://github.com/modelica/fmi-standard/blob/master/CONTRIBUTING.adoc#building-the-specification-document).
* [ ] [Re-generated schema figures](https://github.com/modelica/fmi-standard/blob/master/CONTRIBUTING.adoc#changing-the-xsd-schemas).
* [ ] [Linted the documents](https://github.com/modelica/fmi-standard/blob/master/CONTRIBUTING.adoc#building-the-specification-document).

Add your [meaningful description](https://github.com/modelica/fmi-standard/blob/master/CONTRIBUTING.adoc#bug-reports) here and reference any related issues with `#<issue>`.

### CODE REVIEW

File: docs/2_4_common_schema.adoc [hunk 1]
```diff
@@ -449,17 +449,17 @@ _Note that `Hz` is typically used as `Unit.name` for a frequency quantity, but i
 
 _The `<BaseUnit>` definitions can be utilized for different purposes (the following application examples are optional and a tool may also completely ignore the `Unit` definitions):_
 
-Signal connection check::
+Variable connection check::
```
Comment (chrbertsch) [inline docs/2_4_common_schema.adoc:452]: "Variable connection" sounds a bit strange. 
What about: "Unit check when connecting variables of different FMUs"

File: docs/2_4_common_schema.adoc [hunk 2]
```diff
@@ -449,17 +449,17 @@ _Note that `Hz` is typically used as `Unit.name` for a frequency quantity, but i
...
 +
-_When two signals v1 and v2 are connected together, and on at least one of the signals no `<BaseUnit>` element is defined, then the connection equation "v2 = v1" holds (if v1 is an <<output>> of an FMU and v2 is an <<input>> of another FMU, with `fmi3Get{VariableType}` the value of v1 is inquired and used as value for v2 by calling `fmi3Set{VariableType}`)._
+_When two variables v1 and v2 are connected together, and on at least one of the variables no `<BaseUnit>` element is defined, then the connection equation "v2 = v1" holds (if v1 is an <<output>> of an FMU and v2 is an <<input>> of another FMU, with `fmi3Get{VariableType}` the value of v1 is inquired and used as value for v2 by calling `fmi3Set{VariableType}`)._
```
Comment (chrbertsch) [inline docs/2_4_common_schema.adoc:454]: Suggestion: "When an output of an FMU is connected with an input of another FMU,  and on at least one of these variables no `<BaseUnit>` element is defined, ..."

**Review by chrbertsch — COMMENTED** [review]
Looks good, two small changes suggested

File: docs/2_4_common_schema.adoc (hunk 1 above)
Comment (andreas-junghanns) [reply docs/2_4_common_schema.adoc:452]: Agreed and changed.

**Review by chrbertsch — APPROVED** [review]

File: docs/2_4_common_schema.adoc (hunk 2 above)
Comment (andreas-junghanns) [reply docs/2_4_common_schema.adoc]: I rewrote this entire sentence, please check! Thank you!

**Pull request merged**